fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1202,13 +1204,18 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 
 	self->commitStartTime = now();
 	pProxyCommitData->lastStartCommit = self->commitStartTime;
+	std::unordered_map<uint16_t, Version> tpcvMap;
+	if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
+		tpcvMap = self->tpcvMap;
```

Comment:
Why is the tpcvMap getting copied here?

Reply:
fixing. it should defined as an optional and only set if the knob is set.